src/Controllers/Api/ShiftsController.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Controllers\Api;
+
+use Engelsystem\Http\Request;
+use Engelsystem\Http\Response;
+use Engelsystem\Models\Room;
+use Engelsystem\Models\Shifts\NeededAngelType;
+use Engelsystem\Models\Shifts\Shift;
+use Illuminate\Database\Eloquent\Collection;
+
+class ShiftsController extends ApiController
+{
+    public function entriesByRoom(Request $request): Response
+    {
+        $roomId = (int) $request->getAttribute('room_id');
+        /** @var Room $room */
```

Comment:
Can you add `findOrFail` and `shifts` to [the `Room` model doc](https://github.com/engelsystem/engelsystem/blob/main/src/Models/Room.php#L15)?
Then it will work everywhere and inline type hints should not be required.

Reply:
`findOrFail` is part of the `vendor/illuminate/database/Eloquent/Builder.php` and thus should not be needed to be addressed explicitly and shifts is already there?